content/pages/index.html

```diff
@@ -248,6 +248,29 @@ <h3 id="heading">Create IEEE 829 or free text Test Plan</h3>
                 changeDirection(direction);
                 setTimeout(function() { rollImage(direction); }, 50);
             });
+
+            /* Navigation */
+            var mobileBtnOpen = document.getElementsByClassName('open-menu')[0],
+                mobileBtnClose = document.getElementsByClassName('close-menu')[0],
+                mobileNav = document.getElementsByClassName('mobilenav')[0];
+                mobileLinks = document.querySelectorAll('.mobilenav a[href^="#"]');
+            for(let i = 0; i < mobileLinks.length; i++){
+                mobileLinks[i].addEventListener('click', function(){
+                    closeMenu();
+                });
+            }
+            mobileBtnOpen.addEventListener('click', function(){
+                openMenu();
+            });
+             mobileBtnClose.addEventListener('click', function(){
+                closeMenu();
+            });
+            function closeMenu(){
+                mobileNav.style.top = '-100%';
+            }
+            function openMenu(){
+                mobileNav.style.top = 0;
+            }
```

Comment:
This entire script needs to go into its own static file and be loaded from the base template. Currently the menu works on the index page but not on any other pages.